integration_test/test/test_helper.exs

```diff
@@ -0,0 +1,11 @@
+# Copy _build/test to _build/dev so it only has to be compiled once
+File.rm_rf!(Path.expand("../_build/dev", __DIR__))
+
+File.cp_r!(
+  Path.expand("../_build/test", __DIR__),
+  Path.expand("../_build/dev", __DIR__)
+)
+
+
+ExUnit.configure(max_cases: 2, timeout: 180_000)
```

Comment:
Won't this be too few for a machine with 16 cores? :)

Reply:
Good catch. That was a remnant of the previous integration test setup. 😅

I just pushed an update.